apps/backend/test/constants/groups-test.constant.ts

```diff
@@ -2,15 +2,18 @@ import {CreateGroupDto} from '../../src/groups/dto/create-group.dto';
 
 export const GROUP_1 = {
   name: 'Heimdall Group',
-  public: true
+  public: true,
+  desc: ''
```

Comment:
one of these groups should not have 'desc' set cause we want to test the optional thing right?

Reply:
No - the createGroupDTO still needs a desc to be passed in even though the decorator says @IsOptional. It is similar to the "public" field, which should have a default passed in in the first place.